js/ui/control/scale.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+var util = require('../../util/util');
+var Control = require('./control');
+var DOM = require('../../util/dom');
+
+module.exports = Scale;
+
+function Scale(options) {
+    util.setOptions(this, options);
+}
+
+Scale.prototype = util.inherit(Control, {
+    options: {
+    position: 'bottom-left'
+    },
+
+    onAdd: function(map) {
+        var className = 'mapboxgl-ctrl';
+
+        var container = this._container = DOM.create('div', className + '-group', map.getContainer());
```

Comment:
I think the original intent of the `mapbox-ctrl-group` class was to contain controls that require multiple HTML elements. It hasn't been used properly everywhere. 

In this case, we can get by without this container HTML element. Take a look at the [`Attribution` control for an example](https://github.com/mapbox/mapbox-gl-js/blob/Add-scale-control/js/ui/control/attribution.js#L53-L54) 😄